UI/View.js

```diff
@@ -0,0 +1,168 @@
+      oldPost.firstElementChild.setAttribute('src', newPost._photoLink);
+      oldPost.querySelector('.description span').textContent = newPost._descriprion;
+      oldPost.querySelector('.description .tags').textContent = newPost._hashtags.join(',');
+    }
+  }
+
+  doLike(id, flag){
+
+    let post = document.getElementById(id);
+    let like = post.querySelector('#like'); 
+    like.firstElementChild.setAttribute('src','img/like.png');
+    like.style.background = "hsla(307, 29%, 56%, 0.7)";
+    if(flag){      
+      let like = post.querySelector('#like');
+      like.firstElementChild.setAttribute('src','img/like_red.png');
+      like.firstElementChild.style.background = "hsla(307, 29%, 56%, 0.7)";
+      
+    }
+    
+  }
+
+  setAuthorized(user) {
+    let headerAct = document.getElementById('username');
+    let userName = user && user.getUserName();
+    headerAct.textContent = userName || 'Anonim';
+    document.body.classList.toggle('is-auth', user.isUserName());
+  }
+
+  login(){
+    const log = document.getElementById('login');
+    log.innerHTML = `
+    <label >
+        <div  class = "font"> Login </div>
+        <input type="text" id ="user-login" class = "inputs" placeholder = "Input login" autocomplete="off" />
+    </label>
+            
+    <label>
+        <div class="font">Password</div>
+        <input type="text" id="user-password" class = "inputs" placeholder = "Input Password" autocomplete="off"/>
+    </label>
+    `;
+  }
+
+  error(){
+    const err = document.getElementById('error');
+    err.innerHTML = `
+    <div class = "font error">
+      <img  src="img/err.jpg" class = "error" alt = 'Invalid photo'>
+    </div>
+    <h3 class = "font error">Input Error</h3>
+    <h3 class = "font error">Please, try again...</h3> 
+    `;
+  }
+
+  pageAddPost(user){
+    const post = document.getElementById('btn-add');
+    post.innerHTML = `
+    <div class = "font">Your photoPost</div>
+    <label >
+        <div  class = "font"> Photo </div>
+        <input type="text" id ="photo" class = "inputs" placeholder = "Input img/_.jpg" autocomplete="off" />
+    </label>
+    <label >
+        <div  class = "font"> Description </div>
+        <input type="text" id ="decr" class = "inputs" placeholder = "Input decription" autocomplete="off" />
+    </label>
+    <label >
+        <div  class = "font"> HashTags </div>
+        <input type="text" id ="hash" class = "inputs" placeholder = "Input hashTags" autocomplete="off" />
+    </label>
+    <div class = "font">Username:  ${user.getUserName()}</div>
+    <button class="btn btn-add-post" id = "btn-add">
+            add post
+    </button>
+    `;
+  }
+
+  pageEditPost( oldPost, user){
+    const editPost = document.getElementById('dia-edit');
+    editPost.innerHTML = `
+    <div class = "font">Edit photoPost</div>
+    <label >
+        <div  class = "font"> New Photo </div>
+        <input type="text" id ="new-photo" class = "inputs" value = ${oldPost._photoLink} >
```

Comment:
To fill the form you can just select the definet parts of "edit" (or "add" form ) and just set the values.
It will be much shorter